web/views/layout_view.ex

```diff
@@ -11,4 +11,14 @@ defmodule CoursePlanner.LayoutView do
   def get_program_name do
     Settings.get_value("PROGRAM_NAME")
   end
+
+  def render_user_role_based_navbar(%{assigns: %{current_user: current_user}} = conn) do
+    case current_user.role do
```

Comment:
although i still have to handle the case of current_user role being nil or not being in the list